docker-compose-tests.yml

```diff
@@ -5,8 +5,6 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.eas-api
```

Comment:
If the tests are being run inside a docker container, how is it built? i.e. how is the base image referenced?

Reply:
There's a new repository which should cover that coming, but these changes need to be in first